EnglishFilter/sections/css_extended.txt

```diff
@@ -1937,6 +1937,8 @@ imgrum.org#?#.module_blog_page > .blogpost_user_meta[-ext-has=">.adsbygoogle"]
 cutestat.com#?#.stat_details > div.marginBottom_10 > div.stat_half_left[-ext-has="> div > script:contains(window._mNHandle.queue.push)"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/6638
 rarbg2020.org,rarbgproxied.org,rarbg.to,rarbg-to.proxydude.xyz,rarbg-to.pbproxy.red,rarbg.unblocked.lol,rarbg.pbproxy2.co#?#td[align="justify"] > div > table[width="100%"][class="lista"] > tbody > tr:has(tr > td + td[style*="height:"])
+! https://github.com/AdguardTeam/AdguardFilters/issues/127746
+compstudio.in#$#.adtester-container { position: absolute !important; top: -9999px !important; }
```

Comment:
Where do you need the rule? The elements are already blocked.

Reply:
It removes blank spaces between paragraphs (these do not always appear):

![image](https://user-images.githubusercontent.com/57285466/186366993-c7de0207-c779-4c99-9aac-26289c0eed01.png)

Reply:
You know, it's hidden on my end by some generic rules, but maybe this is a rare case.
Please move this rule with task comment to `general_extensions` file.

Reply:
I did it, I hope everything is fine

Reply:
Yep, thank you;)